workflows/service_executor.go

```diff
@@ -0,0 +1,28 @@
+package workflows
+
+import (
+	"github.com/stelligent/mu/common"
+)
+
+// NewServiceExecutor create a new workflow for executing a command in an environment
+func NewServiceExecutor(ctx *common.Context, environmentName string, service string, command string) Executor {
+
+	workflow := new(environmentWorkflow)
+
+	return newWorkflow(
+		workflow.serviceTaskExecutor(environmentName, service, command, ctx.TaskManager),
```

Comment:
need another step in workflow to lookup the serviceName from `mu.yml` if none is provided.